Magento_Theme/layout/default.xml

```diff
@@ -6,5 +6,14 @@
                 <argument name="logo_width" xsi:type="number">185</argument>
             </arguments>
         </referenceBlock>
+
+        <referenceContainer name="footer">
+            <block class="Magento\Framework\View\Element\Template" name="custom.footer" template="Magento_Theme::custom_footer.phtml">
+                <arguments>
+                    <argument name="custom_footer_text" xsi:type="string">My first footer text!</argument>
```

Comment:
When you use text directives that have to be shown in templates from XML and that text has to be displayed for a customer, don't forget to translate it. All the text that is displayed for customers have to be wrapper to translation functions.
In XML use the attribute
`translate="true"`